ducktape/tests/runner_client.py

```diff
@@ -157,7 +158,7 @@ def run(self):
     def setup_test(self):
         """start services etc"""
         self.log(logging.INFO, "Setting up...")
-        self.test.setUp()
+        self.test.setup()
```

Comment:
I attempted (and perhaps succeeded?) to make this backwards compatible:

- If `setUp` is overridden, internally we call `setup`, which will call the user's `setUp`
- If `setup` is overridden, internally we will call the user's `setup` method